src/Microsoft.Identity.Web.OWIN/AppBuilderExtension.cs

```diff
@@ -187,6 +203,8 @@ public static IAppBuilder AddMicrosoftIdentityWebApp(
 
                             if (clientInfoFromServer != null && clientInfoFromServer.UniqueTenantIdentifier != null && clientInfoFromServer.UniqueObjectIdentifier != null)
                             {
+                                RejectInternalClaims(context.AuthenticationTicket.Identity, clientInfoFromServer.UniqueTenantIdentifier, clientInfoFromServer.UniqueObjectIdentifier);
```

Comment:
Shouldn't this match the logic above (should call Reject for tenant ID , then OID)? Why is the second parameter `clientInfoFromServer.UniqueTenantIdentifier` when it's supposed to be a claim type.

Reply:
Agreed, the logic should be like the other if statement. updated this to match